systers_portal/templates/meetup/snippets/user-cell.html

```diff
@@ -0,0 +1,15 @@
+{% load staticfiles %}
+
+{% if user.profile_picture.name and user.profile_picture.name != "False" %}
+  <a href="{{ user.get_absolute_url }}">
+    <img src="{{ user.profile_picture.url }}" class="profile-pic img-square img-responsive img-wh-100" alt="{{ user }} profile picture"/>
+  </a>
+{% else %}
+  <a href="{{ user.get_absolute_url }}">      
+    <img src="{% static 'img/default.png' %}" class="profile-pic img-square img-responsive img-wh-100"
```

Comment:
Yes, I am also unable to find bootstrap class `img-square` on their site, there offer only 3 image shapes and square is not one of them. Sorry for this.

So finally we don't need any class other than `img-responsive`.